src/CustomOscillatorType.ts

```diff
@@ -0,0 +1,3 @@
+type CustomOscillatorType = 'sine' | 'square' | 'sawtooth' | 'triangle' | 'piano';
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is there no way to have anything more solid here? Maybe something like OscillatorType | PIANO_CONST ?

Just an idea, I do not have the answer to this, I just don't like raw strings in such a vital place.

Reply:
Another file feels overkill, I was thinking of a CONST that could be called here and in other files, but that is just me being a PHP developper, I don't think we need to code defensively in this context